src/pages/LoginPage.tsx

```diff
@@ -75,43 +75,58 @@ const LoginPage = () => {
-                />
-                <Field
-                  name='password'
-                  component={Password}
-                  minSymbol={PASSWORD.minLength}
-                  maxSymbol={PASSWORD.maxLength}
-                  isConfirm={false}
-                  error={touched.password ? errors.password : undefined}
-                />
+                <Field name='email'
+                       component={Email}
+                       error={touched.email ? errors.email : undefined}/>
+                <Field name='password'
+                       component={Password}
+                       minSymbol={PASSWORD.minLength}
+                       maxSymbol={PASSWORD.maxLength}
+                       isConfirm={false}
+                       error={touched.password ? errors.password : undefined}/>
                 <div className='checkbox'>
-                  <Field
-                    name='remember'
-                    type='checkbox'
-                    id='remember'
-                    onClick={() => {
-                      setIsChecked(!isChecked)
-                    }}
-                    className={isChecked ? 'checked' : 'unchecked'}
-                  />
-                  <label
-                    htmlFor='remember'
-                    className='labelCheckbox'
-                  >
+                  <Field name='remember'
+                         type='checkbox'
+                         id='remember'
+                         onClick={() => {
+                           setIsChecked(!isChecked)
+                         }}
+                         className={isChecked ? 'checked' : 'unchecked'}/>
+                  <label htmlFor='remember'
+                         className='labelCheckbox'>
                     Stay logged in
                   </label>
                 </div>
-                <Button
-                  buttonType={'submit'}
-                  buttonText={'Sign in'}
-                />
-                <Link to={'/forgotPassword'} className='passwordLink'>
+                <Button buttonType={'submit'}
+                        buttonText={'Sign in'}
+                        className={'button'}/>
+                <Link to={'/forgotPassword'}
+                      className='passwordLink link'>
                   Forgot your password?
                 </Link>
+                <Link to={'/users/sign_in/phone_number'}
+                      className='buttonLink'>
+                  Continue by phone number
+                </Link>
+                <a href='https://google.ru'
+                   className='buttonLink'>
+                  Continue with Google
+                </a>
+                <a href='https://facebook.com'
+                   className='buttonLink'>
+                  Continue with Facebook
+                </a>
+                <a href='https://vk.ru'
+                   className='buttonLink'>
+                  Continue with VK
+                </a>
```

Comment:
remove social links (google, facebook, vk) for now - it will be done into the another ticket